features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1877,6 +1877,54 @@ CREATE TABLE IDN_OAUTH_PAR (
 )
 /
 
+CREATE TABLE IDN_ORG_USER_INVITATION (
+	ID INTEGER NOT NULL,
+	INVITATION_ID VARCHAR(255) NOT NULL,
+	CONFIRMATION_CODE VARCHAR(100) NOT NULL,
+	USER_NAME VARCHAR(255) NOT NULL,
+	DOMAIN_NAME VARCHAR(255) NOT NULL,
+	EMAIL VARCHAR(255) NOT NULL,
+	USER_ORG_ID VARCHAR(255) NOT NULL,
+	INVITED_ORG_ID VARCHAR(255) NOT NULL,
+	USER_REDIRECT_URL VARCHAR(1024) NOT NULL,
+	STATUS VARCHAR(10) NOT NULL,
+	CREATED_AT TIMESTAMP NOT NULL,
+	EXPIRED_AT TIMESTAMP NOT NULL,
+	PRIMARY KEY (INVITATION_ID)
+)
+/
+CREATE SEQUENCE IDN_ORG_USER_INVITATION_SEQ START WITH 1 INCREMENT BY 1 NOCACHE
+/
+CREATE TRIGGER IDN_ORG_USER_INVITATION_TRIG NO CASCADE
+            BEFORE INSERT
+            ON IDN_ORG_USER_INVITATION
+            REFERENCING NEW AS NEW
+            FOR EACH ROW MODE DB2SQL
+            BEGIN ATOMIC
+                SET (NEW.ID) = (NEXTVAL FOR IDN_ORG_USER_INVITATION_SEQ);
+            END
+/
+CREATE TABLE IDN_ORG_USER_ROLE_ASSIGNMENT (
```

Comment:
```suggestion
CREATE TABLE IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT (
```
Otherwise this DB table will be interpreted as user role assignment